pyServer/GuestFS.py

```diff
@@ -120,7 +124,7 @@ def launch(self):
         return self.callGF('Launching', ['launch'])
     
     def list_filesystems(self):
-        (out, err) = self.callGF('Listing Filesystems', ['list-filesystems'], True)
+        (out, err) = self.callGF('Listing Filesystems', ['list-filesystems'], True, retryCount=1)
```

Comment:
nit: if you're updating all existing callers to call with retryCount = 1, why not just make that the default value?

Reply:
Making it parameterized, so in future if we want to change the retry count for few commands then it will be easier.